docker-compose.yml

```diff
@@ -12,6 +12,15 @@ services:
     environment:
       - SERVER_PORT=8080
       - SPRING_PROFILES_ACTIVE=dev
-
+  remand-and-sentencing-db:
+    image: postgres:15
+    networks:
+      - hmpps
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=ras-pass
```

Comment:
If you ever end up using the `java_localstack_postgres` executor on CircleCI, the db, username and password have to be the same thing. Just a heads up based on something that tripped me up :)